library/ns1_record.py

```diff
@@ -267,6 +268,14 @@
     # This is handled in NS1 module_utils
     pass
 
+try:
+    import ruamel.yaml as yaml
+except ImportError:
+    raise SystemExit(
+        'The lib ruamel.yaml is required to use ns1_record module. Please'
+        'install it.'
+    )
+
```

Comment:
In the zone diff PR I asked that we set up a `requirements.txt` for this dependency. This won't hurt, but once we have a `requirements.txt` we won't need it.

Reply:
Good point. Ill remove the check in the next code push.